GuessNumberGame/script.js

```diff
@@ -0,0 +1,110 @@
+//Make a web page that generates a secret random number and the user can guess the number until they get it right. For each guess, the game should display whether the answer is higher, lower, or correct.
+
+// Extensions:
+
+// Track how many wrong guesses the user has made so far and display that count
+// Track what numbers the user has already guessed and display those
+// Let the user choose the range of the secret number
+
+// i want to make a game called guess the secret number
+// 1. we have to choose a random number let's say from 1 - 100
+// 2. we let the player input their number and if the number is the secret number the page says "Congratulations, you've guessed the secret number"
+// if the player's input is lower we say "The secret number is higher than this number"
+// if the player's input is higher we say "The secret number is lower than this number"
+// For fun we can 1. Keep track of how many wrong guesses the user has 2. Keep track of the numbers guessed and display them 3. Let the user choose the range of the secret number
+
+//variable for the wrong guesses
+const guesses = document.getElementById('guesses');
```

Comment:
Non-blocking but sometimes it is a good practice to define a naming convention which can help us identify what each variable, constant or property is. For example since this constant represents a DOM element we could name it `guessesElem` or `guessesDiv` if it is a div for example.